### PR TITLE
Created Automapper for Create Post endpoint

### PR DESCRIPTION
**Pull request by en-aleksandar-stojmenovic**

### CODE REVIEW

File: ProjectSetup/Controllers/V1/PostController.cs [hunk 1]
```diff
@@ -48,7 +52,11 @@ public async Task<ActionResult<Post>> Get([FromRoute] Guid postId)
 		[CustomExceptionFilter(typeof(PostBadRequestException), HttpStatusCode.BadRequest)]
 		public async Task<IActionResult> Create([FromBody] CreatePostRequest postRequest)
 		{
-			var post = await _repository.Post.CreatePost(postRequest);
+			Post mappedPost = _mapper.Map<Post>(postRequest);
```
Comment (bronjaharis) [inline ProjectSetup/Controllers/V1/PostController.cs]: please use "var" instead of Post.

File: ProjectSetup/Controllers/V1/PostController.cs [hunk 2]
```diff
@@ -48,7 +52,11 @@ public async Task<ActionResult<Post>> Get([FromRoute] Guid postId)
...
+
+			mappedPost.CreatedBy = Guid.Parse(HttpContext.GetUserId());
```
Comment (bronjaharis) [inline ProjectSetup/Controllers/V1/PostController.cs]: You can edit GetUserId method to return Guid and parse this parameter inside, or create new method which will call GetUserId and return you Guid parameter.

File: ProjectSetup/Repositories/PostRepository.cs [hunk 3]
```diff
@@ -10,20 +10,20 @@
...
-        }
-
-        public async Task<IEnumerable<Post>> FindAllPostsAsync()
-        {
-            return await FindAll().ToListAsync();
-        }
-
-        public async Task<Post> FindPostByIdAsync(Guid id)
-        {
+	public class PostRepository : RepositoryBase<Post>, IPostRepository
```
Comment (bronjaharis) [inline ProjectSetup/Repositories/PostRepository.cs:13]: Please remove spaces or tabs from this file.

File: ProjectSetup/Controllers/V1/PostController.cs (hunk 1 above)
Comment (en-aleksandar-stojmenovic) [reply ProjectSetup/Controllers/V1/PostController.cs]: I've fixed it.

File: ProjectSetup/Controllers/V1/PostController.cs (hunk 2 above)
Comment (en-aleksandar-stojmenovic) [reply ProjectSetup/Controllers/V1/PostController.cs]: I've fixed it.

**Review by bronjaharis — APPROVED** [review]

**Pull request merged**